include/multipass/cli/prompters.h

```diff
@@ -88,6 +90,24 @@ class NewPassphrasePrompter : public PassphrasePrompter
 
     std::string prompt(const std::string& text = "Please re-enter passphrase") const override;
 };
+
+class BridgePrompter : private DisabledCopyMove
+{
+public:
+    explicit BridgePrompter(Terminal* term) : term(term){};
+
+    ~BridgePrompter() = default;
+
+    bool bridge_prompt(std::vector<std::string>& nets_need_bridging) const;
+
+private:
+    BridgePrompter() = default;
+
+    Terminal* term;
+
+    const std::regex yes{"y|yes", std::regex::icase | std::regex::optimize};
+    const std::regex no{"n|no", std::regex::icase | std::regex::optimize};
+};
```

Comment:
Why is this duplicated from `common_cli.h`?

Reply:
Ouch, good catch! I think we discussed this personally. I think at some point naming changed and I left the old and the new names. Fixed.